### PR TITLE
Point to more specific example lines & fix broken links & use relative paths in README

### PR DESCRIPTION
**Pull request by juno-yu**
# PR Checklist:
- [x] Have you read the [CONTRIBUTING.md](https://github.com/crypto-com/cro-nodelib/blob/master/CONTRIBUTING.md)?

some files moved , e.g. hd_wallet -> hdwallet
might be better use relative paths so that things could link correctly in clone repo / non-master branch

### CODE REVIEW

**Comment by calvinlauco** [comment]
Thanks @juno-yu for spotting the outdated documentation. Good call to use relative path.

Regarding the line numbers, we do not specify the line number in the first place because it would be easily broken when we update our test cases.

But I think having line number would help in user readability. Perhaps we should enforce the line # checks after test cases files are updated in the contribution guideline.

Once again thanks for your contribution.

File: README.md [hunk 1]
```diff
@@ -23,31 +23,31 @@ An example is worth a thousand words, we provide code examples of different use
...
-- [Generate random staking address](https://github.com/crypto-com/cro-nodelib/tree/master/integration-tests/address.spec.ts)
-- [Create staking address from private/public key](https://github.com/crypto-com/cro-nodelib/tree/master/integration-tests/address.spec.ts)
+- [Generate random transfer address](integration-tests/address.spec.ts#L6)
+- [Create transfer address from private/public key](integration-tests/address.spec.ts#L17)
+- [Generate random staking address](integration-tests/address.spec.ts#L55)
+- [Create staking address from private/public key](integration-tests/address.spec.ts#L65)
 
 #### View Key
-- [Generate random view key](https://github.com/crypto-com/cro-nodelib/tree/master/integration-tests/viewKey.spec.ts)
+- [Generate random view key](integration-tests/viewKey.spec.ts#L7)
```
Comment (calvinlauco) [inline README.md:32]:
```suggestion
- [Generate random view key](integration-tests/viewKey.spec.ts#L8)
```

@juno-yu Just a very minor changes needed, The line number of the"Generate random view key" test case is pointing to the `Describe` tag.

Comment (juno-yu) [reply README.md]: ok, both done

**Pull request merged**